app/src/components/searchBar/SearchBar.js

```diff
@@ -14,10 +18,45 @@ export default function SearchBar() {
 
   const onSearchSubmit = (e) => {
     e.preventDefault();
+
+    // Check to see if the user is using a browser labelled as Safari
+    const isSafari =
```

Comment:
It's usually not a good idea to check the user agent to detect browsers, would recommend a check like this to see if a browse supports input type date

```
function isDateSupported() {
	const input = document.createElement('input')
	const value = 'a'
	input.setAttribute('type', 'date')
	input.setAttribute('value', value)
	return input.value !== value
};
```